### PR TITLE
 adds neo4j cyphershell command to test prob

### PR DESCRIPTION
**Pull request by YaphetKG**

### CODE REVIEW

File: helm/plater/values.yaml [hunk 1]
```diff
@@ -28,24 +28,33 @@ neo4j:
...
     periodSeconds: 10
     failureThreshold: 3
     timeoutSeconds: 2
-    tcpSocket:
-      port: 7687
+    exec:
+      command:
+        - bash
+        - -c
+        - IFS="/" read -a auth <<< $NEO4J_AUTH ; cypher-shell -u ${auth[0]} -p ${auth[1]} "match (c) return count(C)"
```
Comment (PhillipsOwen) [inline helm/plater/values.yaml:46]: you have a mismatch of case between the (c) and count(C)

File: helm/plater/values.yaml [hunk 2]
```diff
@@ -28,24 +28,33 @@ neo4j:
     failureThreshold: 3
     timeoutSeconds: 2
     periodSeconds: 10
-    tcpSocket:
-      port: 7687
+    exec:
+      command:
+        - bash
+        - -c
+        - IFS="/" read -a auth <<< $NEO4J_AUTH ; cypher-shell -u ${auth[0]} -p ${auth[1]} "match (c) return count(c)"
```
Comment (PhillipsOwen) [inline helm/plater/values.yaml:35]: i am concerned of the impact of this query on the DB when there is a boat load of records.

perhaps there is a less costly query that could be run here. like maybe "match (n:Concept) return 1 limit 1"

File: helm/plater/values.yaml (hunk 1 above)
Comment (YaphetKG) [reply helm/plater/values.yaml:46]: ah thanks , had done that for the test

**Review by PhillipsOwen — CHANGES_REQUESTED** [review]
see comments

File: helm/plater/values.yaml (hunk 2 above)
Comment (YaphetKG) [reply helm/plater/values.yaml:35]: yeah wasn't sure if count would be impactful, thought it was going to be a simple lookup , good call will update the query to something lighter

**Review by PhillipsOwen — APPROVED** [review]

**Pull request merged**